### PR TITLE
details.message should be there

### PR DESCRIPTION
**Pull request by sjvans**
also: with json formatter, the top level message property is `msg`

### CODE REVIEW

File: test/tests/error-handling-prod.test.js [hunk 1]
```diff
@@ -47,20 +47,18 @@ describe('graphql - error handling in production', () => {
       const log = console.warn.mock.calls[0][1] || JSON.parse(console.warn.mock.calls[0][0])
 
       if (cds.env.features.cds_assert) {
-        // REVISIT: message missing, cds.assert also misses some properties like type, element, entity ...
```
Comment (sjvans) [inline test/tests/error-handling-prod.test.js:50]: those properties are not needed. as long as there is a target, the developer can look up the rest.

File: test/tests/error-handling-prod.test.js [hunk 2]
```diff
@@ -166,22 +164,25 @@ describe('graphql - error handling in production', () => {
       expect(response.data.errors[0].extensions.details[0]).not.toHaveProperty('stacktrace') // No stacktrace in production
       expect(response.data.errors[0].extensions.details[1]).not.toHaveProperty('stacktrace') // No stacktrace in production
       const log = console.warn.mock.calls[0][1] || JSON.parse(console.warn.mock.calls[0][0])
-      const msgProperty = log.msg ? 'msg' : 'message'
 
       if (cds.env.features.cds_assert) {
         // REVISIT: message missing, cds.assert also misses some properties like type, element, entity ...
```
Comment (sjvans) [inline test/tests/error-handling-prod.test.js:169]:
```suggestion
```

Comment (sjvans) [inline test/tests/error-handling-prod.test.js:169]: those properties are not needed. as long as there is a target, the developer can look up the rest.

File: test/tests/error-handling-prod.test.js [hunk 3]
```diff
@@ -166,22 +164,24 @@ describe('graphql - error handling in production', () => {
       expect(response.data.errors[0].extensions.details[0]).not.toHaveProperty('stacktrace') // No stacktrace in production
       expect(response.data.errors[0].extensions.details[1]).not.toHaveProperty('stacktrace') // No stacktrace in production
       const log = console.warn.mock.calls[0][1] || JSON.parse(console.warn.mock.calls[0][0])
-      const msgProperty = log.msg ? 'msg' : 'message'
 
       if (cds.env.features.cds_assert) {
-        // REVISIT: message missing, cds.assert also misses some properties like type, element, entity ...
```
Comment (sjvans) [inline test/tests/error-handling-prod.test.js:172]: those properties are not needed. as long as there is a target, the developer can look up the rest.

**Review by SamuelBrucksch — APPROVED** [review]

**Pull request merged**